microsoft-edge/devtools-guide-chromium/memory-problems/memory-101.md

```diff
@@ -23,165 +23,146 @@ ms.date: 07/21/2023
+## Garbage-collection roots
 
-_Garbage-collection roots_ are made up of _handles_ that are created (as either local or global) when making a reference from native code to a JavaScript object outside of the V8 VM.  All such handles can be found within a heap snapshot under **GC roots** > **Handle scope** and **GC roots** > **Global handles**.  Describing the handles in this documentation without diving into details of the browser implementation may be confusing.  Both garbage-collection roots and the handles aren't something you need to worry about.
+_Garbage-collection roots_, or GC roots, are created by the browser when a reference from the browser's native code to a JavaScript object outside of the V8 virtual machine is made. These references are called _handles_. You can find these handles in the **Memory** tool under **GC roots** > **Handle scope** and **GC roots** > **Global handles**.<!-- Can't find this in the Memory tool. Where is it? -->
 
-There are many internal GC roots, most of which aren't interesting for the users.  From the applications standpoint, there are the following kinds of roots:
+There are many internal GC roots, most of which aren't interesting for web developers. From the webpage's standpoint, the following types of GC roots exist:
 
-*  Window global object (in each iframe).  In the heap snapshots, the `distance` field indicates the number of property references on the shortest retaining path from the window.
+*  Window global object (in each iframe). In a heap snapshot displayed in the **Memory** tool, the `distance` field for a window global object indicates the number of property references that on the shortest retaining path from the window.<!-- If this is useful, this needs clarification: what window? The top-level one? Does retaining path mean the path through the graph from the iframe window to the top-level window? -->
 
-*  The document DOM tree, consisting of all native DOM nodes that are reachable by traversing the document.  Not all of the nodes have JavaScript wrappers, but if a node has a wrapper, the node is alive while the document is alive.
+*  The document DOM tree, consisting of all native DOM nodes that are reachable by traversing the document. Not all of the nodes have JavaScript wrappers, but if a node has a wrapper, the node is alive while the document is alive.
 
-*  Sometimes objects are retained by the debug context in the **Sources** tool and the **Console**, such as after Console evaluation.  Create heap snapshots with a cleared **Console** tool and no active breakpoints in the debugger in the **Sources** tool.
+*  Sometimes objects are retained by the debugging context set by the **Sources** or the **Console** tool, for example when evaluating a JavaScript expression in the **Console** tool. To remove these objects from the **Memory** tool, clear the **Console** tool and deactivate breakpoints in the **Sources** tool before recording a heap snapshot.
```

Comment:
```suggestion
*  Sometimes objects are retained by the debugging context that's set by the **Sources** or the **Console** tool, for example when evaluating a JavaScript expression in the **Console** tool. To remove these objects from the **Memory** tool, clear the **Console** tool and deactivate breakpoints in the **Sources** tool before recording a heap snapshot.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which - "Include _that_ even if the sentence is clear without it. It helps to clarify the sentence structure."